float32[8] motorSpeed
int32[8] motorCurrent